client/actions/actions.js

```diff
@@ -33,10 +33,42 @@ export const getUsername = (username, password) => (dispatch) => {
 };
 
 
-export const changeActivePost = (lat, long) => (dispatch, getState) =>{
-
-  console.log(lat, long);
+export const changeActivePost = (incident_id) => (dispatch, getState) =>{
+  
   const allIncidents = getState().map.allIncidents;
-  dispatch({ type: types.CHANGE_ACTIVE_POST, payload: [lat, long], allIncidents: allIncidents });
+  dispatch({ type: types.CHANGE_ACTIVE_POST, payload: incident_id, allIncidents: allIncidents });
+  
+};
+
+export const getCoordinates = () => (dispatch) => {
+  axios.get(`api/incidents/location/NY`)
```

Comment:
if we want the map to display locations in all areas, and not just NY, we can use the `api/incidents` instead
```suggestion
  axios.get(`api/incidents`)
```

Reply:
yep let's do that